.palantir/revapi.yml

```diff
@@ -136,6 +136,14 @@ acceptedBreaks:
       new: "method void com.palantir.atlasdb.cell.api.DdlManager::dropTables(java.util.Set<com.palantir.atlasdb.keyvalue.api.TableReference>)"
       justification: "Adding getDdlManager() to TransactionManager will not break\
         \ existing usage"
+  "0.1101.0":
+    com.palantir.atlasdb:atlasdb-cassandra:
+    - code: "java.method.parameterTypeChanged"
+      old: "parameter void com.palantir.atlasdb.keyvalue.cassandra.TracingQueryRunner::<init>(===org.slf4j.Logger===,\
+        \ java.util.function.Supplier<com.palantir.atlasdb.cassandra.CassandraTracingConfig>)"
+      new: "parameter void com.palantir.atlasdb.keyvalue.cassandra.TracingQueryRunner::<init>(===com.palantir.logsafe.logger.SafeLogger===,\
+        \ java.util.function.Supplier<com.palantir.atlasdb.cassandra.CassandraTracingConfig>)"
+      justification: "internal API change on tracingqueryrunner"
```

Comment:
I checked [sourcegraph](https://sourcegraph.palantir.build/search?q=context:global+TracingQueryRunner&patternType=standard&sm=1&groupBy=repo) and this is only used in atlasdb so we should be safe